vcstool/commands/import_.py

```diff
@@ -190,8 +211,11 @@ def main(args=None, stdout=None, stderr=None):
         path_help='Base path to clone repositories to')
     args = parser.parse_args(args)
     try:
-        repos = get_repositories(args.input)
-    except RuntimeError as e:
+        input_ = args.input
+        if isinstance(input_, request.Request):
+            input_ = request.urlopen(input_)
+        repos = get_repositories(input_)
+    except Exception as e:
```

Comment:
This hides the stacktrace for all exception types - even unexpected ones. Therefore this should only catch specific / expected exception here.

Reply:
done in 29cc83e5badd0dabbee17d6a3aebb9e8f61246bf